tox.ini

```diff
@@ -60,6 +68,11 @@ ignore = E203,E501,W503
 per-file-ignores =
     gitlab/v4/objects/__init__.py:F401,F403
 
+[isort]
+profile = black
+multi_line_output = 3
+force_sort_within_sections = True
```

Comment:
I think that historically this package has sorted straight imports before from-style (also judging by the diff this PR created). It also seems implied in PEP8 (but not mentioned) and other styleguides. Am I missing something here and this other approach is more common now? Otherwise I think I'd personally go with the default :)

Reply:
Again, my personal preference.

As then the imports are ordered by the module name. Not changed depending of if prefaced by `import` or `from`.

When I worked on OpenStack we tended to do it in this style. So it is what I am use to.

Reply:
Ok, thanks :) I think in this case I'd agree with the answers in https://softwareengineering.stackexchange.com/q/418329 and stay with defaults for this one (but use the other type-sort setting above).

It will also make the diff a bit smaller, we already need a `.git-blame-ignore-revs` for all the refactoring we've done in the past year IMO :grin:

Reply:
As a counterpoint: https://docs.openstack.org/hacking/latest/user/hacking.html

OpenStack is probably the largest Python project in the world

Reply:
I'll update this patch later today and remove the `force_sort_within_sections` option.

Reply:
Done! 😊